<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_plant</class>
 <widget class="QDialog" name="new_plant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 170, 0);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>731</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="3">
     <widget class="QSpinBox" name="flowering_suntime_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="growing_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Growing Days</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="flowering_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Flowering Day</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="3">
     <widget class="QSpinBox" name="growing_suntime_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>24</number>
      </property>
      <property name="value">
       <number>12</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="day_temp_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Day T°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="watering_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Humidity (%)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QSpinBox" name="flowering_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>365</number>
      </property>
      <property name="value">
       <number>30</number>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QDoubleSpinBox" name="day_temp_doubleSpinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <double>40.000000000000000</double>
      </property>
      <property name="value">
       <double>22.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QDoubleSpinBox" name="watering_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>50.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="2">
     <widget class="QLabel" name="flowering_suntime_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Flowering Suntime
(in h/day)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="5">
     <widget class="QTextEdit" name="desc_textEdit">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QTextEdit" name="name_textEdit">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);
color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="4">
     <widget class="QDialogButtonBox" name="command_buttonBox">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::RestoreDefaults</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="5">
     <widget class="QLabel" name="desc_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Description</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="name_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Name of the
plant</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QLabel" name="growing_suntime_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Growing Suntime
(in h/day)</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QSpinBox" name="growing_spinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="maximum">
       <number>365</number>
      </property>
      <property name="value">
       <number>60</number>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="night_temp_lab">
      <property name="font">
       <font>
        <family>Piboto Condensed</family>
       </font>
      </property>
      <property name="text">
       <string>Night T°</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QDoubleSpinBox" name="night_temp_doubleSpinBox">
      <property name="maximumSize">
       <size>
        <width>60</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(126, 63, 0);</string>
      </property>
      <property name="maximum">
       <double>40.000000000000000</double>
      </property>
      <property name="value">
       <double>22.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="main_title_lab">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>700</width>
     <height>80</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>700</width>
     <height>80</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Piboto Condensed</family>
     <pointsize>24</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Enter the new plant's properties</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>command_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>new_plant</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>command_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>new_plant</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
